docs/user/guides/jupyter.rst

```diff
@@ -237,21 +237,10 @@ Creating galleries of examples using notebooks
 
 `nbsphinx`_ has support for :doc:`creating thumbnail galleries from a list of Jupyter
 notebooks <nbsphinx:subdir/gallery>`.
-This functionality relies on `Sphinx-Gallery <https://sphinx-gallery.github.io/>`_
-and extends it to work with Jupyter notebooks rather than Python scripts.
+This functionality is inspired by, but doesn't depend on
+`Sphinx-Gallery <https://sphinx-gallery.github.io/>`_.
```

Comment:
I suppose we can just drop this mention of sphinx-gallery too, it's seems more confusing than helpful if we aren't going to explain the difference between the two.

Reply:
Sure, I've removed it in 2205a5323e5a88de65f00de094afaf15febc0e4b.